client/src/main.jsx

```diff
@@ -42,7 +43,11 @@ const router = createBrowserRouter([
       {
         path: "/how-to-use",
         element: <HowToUse />
-      }
+      },
+      {
+        path: "/teacherview",
+        element: <TeacherView />
```

Comment:
For now this is fine, but eventually we want to not have this component in its own page